aspnetcore/host-and-deploy/proxy-load-balancer.md

```diff
@@ -305,6 +305,38 @@ services.Configure<ForwardedHeadersOptions>(options =>
 > [!IMPORTANT]
 > Only allow trusted proxies and networks to forward headers. Otherwise, [IP spoofing](https://www.iplocation.net/ip-spoofing) attacks are possible.
 
+::: moniker range=">= aspnetcore-2.1 <= aspnetcore-2.2"
+
+## .NET Core templates fail in Azure Linux due to missing scheme forwarders
```

Comment:
Could simplify this to Add scheme forwarders for Linux.

Also, this might fit better before Troubleshooting rather than after it, so Troubleshooting remains the last section before Next steps. Or make this an H3 under Troubleshooting and change the heading text to indicate the error text you'd see before adding Forwarded Headers Middleware.